### ISSUE TITLE
INFO_CLICK is not working in cluster maps

### ISSUE THREAD

**Issue by jsasmitha** [mapsplugin/cordova-plugin-googlemaps#2527]
<!--

I'm working with cluster markers , I'm facing issue with info window click event which is not firing. I have shared my code below. 

**I'm submitting a ...**  (check one with "x")
- [] question
- [x] any problem or bug report

**OS:**  (check one with "x")
- [ ] **Android**
- [ ] **iOS**
- [ x] **Browser**

**cordova information:**  (run `$> cordova plugin list`) 

```
 <plugin name="cordova-plugin-whitelist" spec="1.3.3" />
    <plugin name="cordova-plugin-statusbar" spec="2.4.2" />
    <plugin name="cordova-plugin-device" spec="2.0.2" />
    <plugin name="cordova-plugin-splashscreen" spec="5.0.2" />
    <plugin name="cordova-plugin-ionic-webview" spec="^2.0.0" />
    <plugin name="cordova-plugin-ionic-keyboard" spec="^2.0.5" />
    <plugin name="cordova-plugin-datepicker" spec="0.9.3" />
    <plugin name="cordova-plugin-googlemaps" spec="2.4.6">

my packega.json
```
    "@angular/animations": "5.2.11",
    "@angular/common": "5.2.11",
    "@angular/compiler": "5.2.11",
    "@angular/compiler-cli": "5.2.11",
    "@angular/core": "5.2.11",
    "@angular/forms": "5.2.11",
    "@angular/http": "5.2.11",
    "@angular/platform-browser": "5.2.11",
    "@angular/platform-browser-dynamic": "5.2.11",
    "@ionic-native/core": "^4.18.0",
    "@ionic-native/date-picker": "^4.18.0",
    "@ionic-native/google-maps": "^4.15.1",
    "@ionic-native/splash-screen": "~4.17.0",
    "@ionic-native/status-bar": "~4.17.0",
    "@ionic/storage": "2.2.0",
    "cordova-browser": "5.0.4",
    "cordova-plugin-datepicker": "0.9.3",
    "cordova-plugin-device": "^2.0.2",
    "cordova-plugin-googlemaps": "2.4.6",
    "cordova-plugin-ionic-keyboard": "^2.1.3",
    "cordova-plugin-ionic-webview": "^2.3.1",
    "cordova-plugin-splashscreen": "^5.0.2",
    "cordova-plugin-statusbar": "^2.4.2",
    "cordova-plugin-whitelist": "^1.3.3",
    "ionic-angular": "3.9.2",
    "ionic2-rating": "^1.2.2",
    "ionicons": "3.0.0",
    "rxjs": "5.5.11",
    "sw-toolbox": "3.6.0",
    "zone.js": "0.8.26"
```

**Current behavior:**
getting following error:-
polyfills.js:3 Uncaught Error: Listener for one()/addEventListenerOnce() method is not a function
    at Marker.one (BaseClass.js:108)
    at map-view.ts:339
    at t.invokeTask (polyfills.js:3)
    at r.runTask (polyfills.js:3)
    at e.invokeTask (polyfills.js:3)
    at i.isUsingGlobalCallback.invoke (polyfills.js:3)
    at n (polyfills.js:3)

**Expected behavior:**
On click it should show a alert

**Related code, data or error log (please format your code or data):**

 let markerCluster: MarkerCluster = this.map.addMarkerClusterSync({
      markers: data,
      boundsDraw: false,
      icons:[]
    });
    let marker: Marker;
    markerCluster.on(GoogleMapsEvent.MARKER_CLICK).subscribe((params) => {
       marker = params[1];
      marker.setTitle(marker.get("name"));
      marker.setSnippet(marker.get("address"));
      marker.showInfoWindow(); 
        marker.one(GoogleMapsEvent.INFO_CLICK).then(()=>{
          console.log('info click');
        })
    });

**Comment by wf9a5m75** [comment]
Please share your project files on GitHub repository

**Comment by JZubero** [comment]
I am having the same problem.

*Env Info*:
```
   ionic (Ionic CLI)  : 4.5.0 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.1
```

```
basicCluster.on(GoogleMapsEvent.MARKER_CLICK).subscribe(params => {
  // Get marker from action params
  let marker: Marker = params[1];
  
  // Get payload data from marker
  const children: IMarkerItem[] = marker.get('children');
  
  // Remove previously bound listeners
  marker.off(GoogleMapsEvent.INFO_CLICK);
  
  const item = children[0];
  if (children.length === 1) {
    // Show event info window
    marker.setTitle(item.title);
    marker.setSnippet(item.subTitle);
    marker.showInfoWindow();
    
    // Push event detail page
    marker.one(GoogleMapsEvent.INFO_CLICK).then(() => {
      this.navCtrl.push(item.link.page, item.link.params).then(() => marker.hideInfoWindow());
    });
  } else {
    // Show location info window
    marker.setTitle(item.venue);
    marker.setSnippet(`Hier finden ${children.length} Veranstaltungen statt.`);
    marker.showInfoWindow();
    
    // Push event list page
    marker.one(GoogleMapsEvent.INFO_CLICK).then(() => {
      this.navCtrl.push('VenueEventListPage', {
        venue: item.venue,
        markers: children
      }).then(() => marker.hideInfoWindow());
    });
  }
});
```

**Comment by wf9a5m75** [comment]
Please share your project files on GitHub repository, not pasting part of your code

**Comment by wf9a5m75** [comment]
ping

**Comment by SimFerrer** [comment]
Hello,

I have the same problem

My GitHub repository : https://github.com/SimFerrer/cluster_infoclick

**Comment by wf9a5m75** [comment]
Thank you @SimFerrer . The next version of this plugin and `@ionic-native/google-maps` fix this issue.

**Comment by wf9a5m75** [comment]
This bug is fixed in v2.5.0 and `@ionic-native/google-maps v5.0.0-beta.26`
